### DISCUSSION TITLE
How are cases analysed with an old gene panel version handled when a new version is available in Scout?

### DISCUSSION THREAD

**Discussion by 4WGH** [Clinical-Genomics/scout#4423] [Q&A]
![image](https://github.com/Clinical-Genomics/scout/assets/12525201/d5c078e7-fc5a-40ba-8193-781b07d2d208)

I need to check it I understand correctly:
it version 10 used? are all genes in version 10 analyzed even if not present in the clinical master list at the time of the analys?
are genes " present in case panel and not in latest version" like AASS analyzed?

Could you then update in the case window that the defaul panel is v10 and not v9?

Can you write in the report file that version 10 is used?

Is the coverage calculated for version 9 or 10?

Data need to match in documentation.... otherwise there is confusion.....

**Comment by northwestwitch** [comment]
As a general rule, if a case is a runned with a panel, then the analysis is is performed with that panel.

Answering point to point:

> it version 10 used? are all genes in version 10 analyzed even if not present in the clinical master list at the time of the analys?
are genes " present in case panel and not in latest version" like AASS analyzed?

If it says panel v.9, why should panel 10 be used instead? 😆 
If you look for the AASS gene you should find it among the cases's variants, since it's in panel v.9, but perhaps not filtering using that panel in the variantS filters, so you look for the specific gene instead.

To have all the genes present in panel v10 available for your case you should request - if your case is not super old, in that case do a rerun - a reupload specifying that the VCF files used for the upload should change because the panels have changed.

> Can you write in the report file that version 10 is used?

Isn't the info already present in the general report (panel 9, not 10)? Not that the panel is outdated, the version I mean.

> Is the coverage calculated for version 9 or 10?

Panel v.9!

**Comment by dnil** [comment]
Absoultely agree: the analysis is performed with the specified version (9), and the division between research and clinical genes is done based on the same version (9). For a full reanalysis, you are still generally safer withwith a rerun.

Consider the following points though:
- It has been a while since the panel was analysed, and a new panel version is out. On this version, it was decided we no longer want to see some genes (e.g. AASS). The best course of action is to not show them then. You would only conclude you have already decided to leave the gene off the panel.
- Conversely, new phenotypes have been connected to some genes and the panel has been updated with some new disease causing genes. The best we can do in this situation is show as many of those as possible when filtering, given the analysis limitations. Often almost all genes were already on another clinical panel, so they can just be shown.

If you are concerned about the phenotype in relation to some of the removed genes (AASS) consdier analysing with a panel that has that gene and possibly more complete coverage for that range of phenotype (IEM here perhaps).

The panel versions are clearly stated on the coverage reports, defaulting to the version the case was analysed with (9), as Chiara says.

I would consider report this case as having been analysed with version 9, to leave v10 rerun an option. But if you want to communicate exactly what has been done, consider the custom panel reports. From the case panel (say IF 9 😸), select the Case specific extent pdf:
![Screenshot 2024-02-08 at 08 44 19](https://github.com/Clinical-Genomics/scout/assets/758570/7fd015de-5448-4898-bd19-d173d4069339)
![Screenshot 2024-02-08 at 08 43 52](https://github.com/Clinical-Genomics/scout/assets/758570/8ce0e0ee-8f7c-4a08-8ea7-1f3ddfd4c4cf)
which will download a pdf where all the warnings you saw on the filtering page are also documented.

**Comment by 4WGH** [comment]
"If it says panel v.9, why should panel 10 be used instead? 😆"

I ask because there is a  criptic message (as in pic) that user interpret in different ways.

We are still a bit confiused, after the 2 answers we received. 

If a apply the clinical filter will AASS variant appear or not? AASS is taken as an example to understand. If ti state dafault panel 9 I expect in the variant filtering that variants in AASS are shown. 

We send the panel version in the answer, that need to be correct.

There is a reason for panel versioning (=> trakckability and documentation about what has been analyzed)

**Comment by dnil** [comment]
Hi, this issue has been vented a few times since the introduction many years back, and we have reached the same conclusion. I have yet to hear a counterargument. But we are very keen to make it more clear!

Please specify clearly what is not trackable and documented currently? And perhaps also what is cryptic about the warning? How could it be improved?

As for the clinical filter, it is no different from selecting the panel from the other variantS filters. 

Consider ATM for instance and I think you will see clearly what is going on. ATM was for good reason also removed from the panel in v10 in December. It is of course in other clinical panels for the case, e.g. NMD or OMIM-AUTO. 

The case has been analysed with v9 in November. Variants in ATM are analysed (in the pipeline) reagardless if v9 or v10 in this case, as the gene is anyway in the clinical master list from other clinical panels.

When you search for variants for the case, with IF panel directly or indirectly with clinical filter and IF marked default, you will NOT see variants in the removed ATM gene.

You can, if you wish, still see the variants, by explicitly searching for ATM, or selecting an appropriate gene panel (eg NMD, OMIM or say ChiCAP Actionable).

**Comment by 4WGH** [comment]
There has been a discussion what panel version to give in our answer. Which is most correct?

I've though that 9 was more correct, others though that v10 was more near to the real analysis done.

In any case independently if we chose 9 or 10 none is fully correct. this make me unconfortable. If genes in v9 that are removed in version 10 were not hidden from analysis I would fell that at least the analysis with v9  was correctly done and we had a bonus chance to look at some extra variants present in gene added with version 10.
I will report with v9 as you also suggest :-)

thanks

**Reply by northwestwitch** [reply]
You can be comfortable to answer that the analysis has been done with panel v.9. And that is the fully correct answer. Then as you mentioned you can say that you were able to check also other genes that were in other panels (some of which only by chance in the same panel but v.10). I think

**Reply by dnil** [reply]
In general this means analysis is lagging compared to when answers could be out, which makes me uncomfortable.😛 But I know how it is sometimes.

As for the panel version to report when reporting a several months old case that was not rerun, a situational judgement call is in order. If all genes on the new version were indeed on the clinical list, there is little reason to order a rerun. If any of the removed genes are still of interest to the phenotype I would recommend reanalysis with the now phenotype relevant gene panel. And regarding what gene list to give, if neither of the two versions are comfortable, consider using the dynamically generated list which will describe the somewhat complex situation with fidelity.
